core/trie/trie_test.go

```diff
@@ -87,52 +88,296 @@ func TestFindCommonPath(t *testing.T) {
 }
 
 func TestTriePut(t *testing.T) {
+	t.Run("put to empty tree", func(t *testing.T) {
+		RunOnTempTrie(251, func(trie *Trie) error {
+			keyNum, _ := strconv.ParseUint("1101", 2, 64)
+			key := new(felt.Felt).SetUint64(keyNum)
+			val := new(felt.Felt).SetUint64(11)
+			if _, err := trie.Put(key, val); err != nil {
+				t.Errorf("Put() failed")
+			}
+
+			if value, err := trie.Get(key); err != nil || !value.Equal(val) {
+				t.Errorf("key-val not match")
+			}
+			assert.Equal(t, trie.FeltToBitSet(key), trie.rootKey)
```

Comment:
You shouldn't mix and match the use of `t.Error`/`t.Fatal` with `assert`/`require`. You should also be logging the error not just what failed.

```suggestion
			_, err := trie.Put(key, val)
			require.NoError(t, err)

			value, err := trie.Get(key);
			require.NoError(t, err)
			
			assert.Equal(t, val, value)
			assert.Equal(t, trie.FeltToBitSet(key), trie.rootKey)
```

Reply:
I sloved this comment at [here](https://github.com/NethermindEth/juno/pull/539/commits/f01c7d687383d9c87aefb6e4ad85b47241b9a229).